backend/routes/inviteContactsRouter.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
+import { body } from 'express-validator'
+import { handleInputErrors } from '../modules/middleware'
+import InviteContactController from '../controllers/inviteContacts'
+const inviteContactsRouter = express.Router();
+
+inviteContactsRouter.post('/',
+  body('planID').isString().notEmpty(),
+  body('message').isString().notEmpty(),
+  body('selectedContacts')
+    .custom((value: any[]) => {
+      return value.every((contact) => {
+        return 'id' in contact &&
+          'firstName' in contact &&
+          'ownerID' in contact
+      })
```

Comment:
this is fine, but you can also use `Object.hasOwn()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn